src/sql/src/plan/statement/ddl.rs

```diff
@@ -377,8 +382,8 @@ pub fn plan_create_source(
             let encoding = get_encoding(format, envelope, with_options_original)?;
 
             let mut connector = KafkaSourceConnector {
-                addrs: broker.parse()?,
                 topic: topic.clone(),
+                addrs: broker.parse()?,
```

Comment:
It seemed like the other changes of this form were sorting the fields into a more consistent order (connector then the rest), but this one is moving in the opposite direction?